Dockerfile

```diff
@@ -37,6 +37,6 @@ FROM alpine
 ENV APP_PORT=49988
 EXPOSE $APP_PORT
 
-COPY --from=builder /device-random/cmd /
+COPY --from=builder /go/src/github.com/edgexfoundry/device-random/cmd /
```

Comment:
The update to this `COPY` directive was to fix an _existing_ error encountered when running `make docker`, as follows:
```
   ---> 4e7ffef6d1b4
  Step 14/16 : COPY --from=builder /device-random/cmd /
  COPY failed: stat /var/lib/docker/overlay2/6e093e7a8a64ec10212b71126e6b0171391e3527921bbe5b17bb46d409d77665/merged/device-random/cmd: no such file or directory
  make: *** [docker_device_random_go] Error 1
```